pkg/controller/replicaset/replica_set_test.go

```diff
@@ -1373,3 +1374,254 @@ func TestRemoveCondition(t *testing.T) {
 		}
 	}
 }
+
+func TestSlowStartBatch(t *testing.T) {
+	err := fmt.Errorf("fake error")
```

Comment:
Call this `fakeErr` so we don't accidentally overwrite it anywhere with normal `err`.